scripts/functions.sh

```diff
@@ -55,3 +57,35 @@ for rc_file in openstack-release os-release lsb-release redhat-release; do
     source "/etc/${rc_file}"
   fi
 done
+
+## Functions -----------------------------------------------------------------
+
+function apt_artifacts_available {
+
+  CHECK_URL="${HOST_RCBOPS_REPO}/apt-mirror/integrated/dists/${RPC_RELEASE}-${DISTRIB_CODENAME}"
+
+  if curl --output /dev/null --silent --head --fail ${CHECK_URL}; then
+    return 0
+  else
+    return 1
+  fi
+
+}
+
+function configure_apt_sources {
+
+  # Replace the existing apt sources with the artifacted sources.
+
+  sed -i '/^deb-src /d' /etc/apt/sources.list
```

Comment:
in order to be able rewind artifacts the old file should be backed up first. Something simple like 

``` shell
if [[ ! -f "/etc/apt/sources.list.original" ]]; then
  cp /etc/apt/sources.list /etc/apt/sources.list.original
fi
```